install.sh

```diff
@@ -0,0 +1,877 @@
+#!/bin/sh
+
+set -e
+
+if [ "${DEBUG}" = 1 ]; then 
+    set -x
+fi
+
+# Usage:
+#   curl ... | ENV_VAR=... sh -
+#       or
+#   ENV_VAR=... ./install.sh
+#
+# Example:
+#   Installing a server without traefik:
+#     curl ... | INSTALL_RKE2_EXEC="--disable=traefik" sh -
+#   Installing an agent to point at a server:
+#     curl ... | RKE2_TOKEN=xxx RKE2_URL=https://server-url:6443 sh -
```

Comment:
so in rke2 the supervisor port is set to 9345 by default, so this command will fail if its using 6443